v3/plugins/ms-vscode/typescript/1.49.3/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
```

Comment:
Now that #721 is closed, we do not need this file, correct @benoitf ?

Reply:
correct, once aliases and entry are added in che-theia-plugins.yaml ( what this PR dors) we can remove it

Reply:
@tsmaeder please remove the meta.yaml changes and we should be good to go.